third_party/webcomponentsjs/ShadowCSS.js

```diff
@@ -236,7 +236,7 @@ export const ShadowCSS = {
       scoped = parts.map(function(p) {
         // remove :host since it should be unnecessary
         var t = p.trim().replace(polyfillHostRe, '');
-        if (t && (splits.indexOf(t) < 0) && (t.indexOf(attrName) < 0)) {
+        if (t && (!splits.includes(t)) && (!t.includes(attrName))) {
```

Comment:
The `t.includes` is `String#includes`. `String#includes` is not supported in IE.